libraries/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -687,12 +705,15 @@ TEST_GROUP_RUNNER( MQTT_Unit_API )
     RUN_TEST_CASE( MQTT_Unit_API, DisconnectAlreadyDisconnected );
     RUN_TEST_CASE( MQTT_Unit_API, PublishQoS0Parameters );
     RUN_TEST_CASE( MQTT_Unit_API, PublishQoS0MallocFail );
+    RUN_TEST_CASE( MQTT_Unit_API, PublishSyncQoS0WithNetworkFailure );
```

Comment:
Nit: `PublishQoS0SyncWithNetworkFailure` seems like it would fit better with the other test names

Reply:
Changed